src/pages/schedule-page.js

```diff
@@ -133,7 +138,7 @@ export default function () {
                             ? data.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
                             : data
                         ).map((row) => (
-                            <TableRow key={row.id}>
+                            <TableRow key={row.id} onClick={handleTableRowClick}>
```

Comment:
Don't we have to send `row.moduleCode`  to `handleTableRowClick` function?

Reply:
Yes. Ideally id has to be passed. But none of these navigations are logical. Those are for the demonstration purpose. I hope we can add the logical components when we are connecting the backend "at some point" right? :)